SMUser/schema.json

```diff
@@ -30,23 +30,28 @@
           "type": "string",
           "description": "Property.  Model:'https://schema.org/Text'. Units:'No unit'. The username of the SMUser."
         },
-	  "createdPost": {
-          "type": "string",
-          "format": "uri",
-          "description": "Relationship. The ID of the post that the SMUser created."
+	  "createdPosts": {
+      "type": "array",
+	  "description": "Relationship. The IDs of the post that the SMUser created.",
+	  "items": {
+	  "type": "string",
+          "format": "uri"
+	  }
         },
 	  "isMentionedBy": {
-          "type": "string",
-          "format": "uri",
-          "description": "Relationship. The ID of a post that mentions the SMUser."
+	  "type": "array",
+	  "description": "Relationship. The IDs of the posts that mention the SMUser.",
+	  "items": {
+	  "type": "string",
+                  "format": "uri"
+	  }
+      } 
         }
       }
-    }
   ],
   "required": [
     "id",
     "type",
-    "userId",
     "platform",
     "userName"
   ]

```

Comment:
final remark, please validate that the schema.json validates example.json. 
this tool would be helpful https://www.jsonschemavalidator.net/

Reply:
Using the above tool, all 5 entities have been validated, their respective schema.json validates example.json .